pandas/_libs/window.pyx

```diff
@@ -32,11 +32,10 @@ cdef double NaN = <double> np.NaN
 cdef inline int int_max(int a, int b): return a if a >= b else b
 cdef inline int int_min(int a, int b): return a if a <= b else b
 
-from util cimport numeric
+from util cimport numeric, sqrt
 
 cdef extern from "../src/headers/math.h":
     int signbit(double) nogil
```

Comment:
prob worthwhile adding any of the imports from src/headers/math here as well (prob not very many)

Reply:
> prob worthwhile adding any of the imports from src/headers/math here as well (prob not very many)

Agreed; its been added to the src todo list.  Need to move on for now.